fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2870,6 +2870,7 @@ public class Config extends ConfigBase {
     public static int profile_info_reserved_num = 500;
 
     /**
+     * Deprecated
```

Comment:
add `comment = "deprecated"` to the confVar so that `admin show frontend config ...` will show the comment and the deprecation info can be visible to the user.

Reply:
Currently we can't delete this variable, direct deletion may result in an upgraded FE needing to delete this configuration in the config file before it can start.

Reply:
not necessary to delete the session variable, but the configvar can be annotated as deprecated

```
    @ConfField(mutable = true, comment = "Deprecated")
    public static int load_profile_info_reserved_num = 500;
```

Before the change
```
mysql> admin show frontend config like 'load_profile_info_reserved_num';
+--------------------------------+------------+-------+------+-----------+---------+
| Key                            | AliasNames | Value | Type | IsMutable | Comment |
+--------------------------------+------------+-------+------+-----------+---------+
| load_profile_info_reserved_num | []         | 500   | int  | true      |         |
+--------------------------------+------------+-------+------+-----------+---------+
1 row in set (0.00 sec)
```

After the change
```
mysql> admin show frontend config like 'load_profile_info_reserved_num';
+--------------------------------+------------+-------+------+-----------+------------+
| Key                            | AliasNames | Value | Type | IsMutable | Comment    |
+--------------------------------+------------+-------+------+-----------+------------+
| load_profile_info_reserved_num | []         | 500   | int  | true      | Deprecated |
+--------------------------------+------------+-------+------+-----------+------------+
1 row in set (0.02 sec)
```